schunk_lwa4d/scripts/move_circ.py

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env python
+import math
+import rospy
+
+from geometry_msgs.msg import Pose
+from cob_cartesian_controller.msg import Profile
+from simple_script_server.simple_script_server import simple_script_server
```

Comment:
please add the dependency to `cob_script_server` 
https://github.com/ipa320/schunk_robots/blob/indigo_dev/schunk_lwa4d/package.xml